interpreter/exec/eval.ml

```diff
@@ -72,10 +72,12 @@ and admin_instr' =
   | Label of int * instr list * code
   | Frame of int * frame * code
   | Handler of int * catch list * code
-  | Handle of (tag_inst * idx) list option * code
+  | Handle of (tag_inst * handler_target) list option * code
```

Comment:
We should come up with more telling names for distinguishing these. Can't think of anything good off-hand, though.

Reply:
`ExnHandler` and `CtrlHandler`?

Reply:
Yeah, ideally it would be shorter, single word. Perhaps `Catch` and `Control`?

Reply:
Yes, but then we need to disambiguate `Catch`.

Reply:
Maybe we should use `Prompt` rather than `Control`.

Reply:
Perhaps we should think a little more about this. I will merge this PR soon.